collaborative/IOU/vulnerability_assessment.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "90ef497a-dcb7-4b23-aa33-2b7690294144",
+   "metadata": {},
+   "source": [
+    "# Vulnerability Assessment Pilot\n",
+    "This notebook demonstrates on-going development of climate adaptation vulnerability assessment (CAVA) support using climate data in the Analytics Engine. \n",
+    "\n",
+    "To execute a given 'cell' of this notebook, place the cursor in the cell and press the 'play' icon, or simply press shift+enter together. Some cells will take longer to run, and you will see a [$\\ast$] to the left of the cell while AE is still working.\n",
+    "\n",
+    "**Intended Application**: As a user, I want to **<span style=\"color:#FF0000\">access climate projections data for my vulnerability assessment report</span>** by:\n",
+    "1. Retrieve data metrics required for planning needs\n",
+    "\n",
+    "**Runtime**: With the default settings, this notebook takes approximately **less than 10 minutes** to run from start to finish. Modifications to selections may increase the runtime. <br>*This notebook is currently in progress, runtime will change as improvements and further analyses are added.*\n",
+    "\n",
+    "### Step 0: Set-up\n",
+    "\n",
+    "First, we'll import the Python library [climakitae](https://github.com/cal-adapt/climakitae), our AE toolkit for climate data analysis, along with this specific functions from that library that we'll use in this notebook, as well as any other necessary Python libraries to aid in analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be3d6530-a94d-416e-81a3-2fac5e79bf72",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import climakitae as ck\n",
+    "import pandas as pd\n",
+    "\n",
+    "from climakitae.explore.vulnerability import cava_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "153b50c3-5e8c-4f4c-b7e2-1839ae0c462f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Step 1: Import locations\n",
+    "Now we'll read in point-based locations that we want to retrieve data for. For custom inputs, there are two options: (1) Input a single pair of latitude - longitude values; and (2) Import a csv file of locations that will each run. In the code below we show what each option looks like. \n",
+    "\n",
+    "Functionality to assess over a gridded area (region) is in the works as well!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f86bed4b-a58d-4d9b-9a06-adfb5fb89717",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## select a single custom location\n",
+    "# your_lat = LAT\n",
+    "# your_lon = LON"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c435c50f-dec5-4440-9bd6-e5207a317f92",
+   "metadata": {},
+   "source": [
+    "To import your own custom locations, we recommend putting your csv file in the same folder as this notebook for ease:\n",
```

Comment:
A dedicated folder for such files would be nice to have, similarly for exported data, to keep the folder organized. 

Showing example_file.csv as an example may also help.